### DISCUSSION TITLE
BUG NOT WORKING ON PHONE ANDRIOD

### DISCUSSION THREAD

**Discussion by Comfy242** [NillionNetwork/discussions#57] [Bugs]
I realised phone users can install the nillion program as it requires desktop.  nillion should as well bring in innovation on how phone users can run the application files without having to use a computer

**Comment by oceans404** [comment]
The CLI dev tool binaries are meant to be downloaded by developers to create blind apps and tools on Nillion, not directly by users.

https://docs.nillion.com/nillion-sdk-and-tools
